config/application.rb

```diff
@@ -29,5 +29,7 @@ class Application < Rails::Application
     config.time_zone = 'London'
 
     config.active_record.raise_in_transactional_callbacks = true
+    config.unsubscribe_url =
+      '<https://www.prisonvisits.service.gov.uk/unsubscribe>'
```

Comment:
This one needs a rethink.  @dan-palmer and I just looked at it on the old app and it doesn't give you the option of unsubscribing. It only tells you why you're receiving the email.  It also has a strange workflow in that it is being served by the existing app, but is also extracted as a static page into `/public` by a rake task.  Dan thinks this is so it can be served external to the app.  

I'll raise a separate ticket for this now.